README.md

```diff
@@ -59,6 +60,12 @@ Fully working examples of how to use these components are in `/example-crates`:
 [`electrum-client`]: https://docs.rs/electrum-client/
 [`bdk_chain`]: https://docs.rs/bdk-chain/
 
+## Dependencies
+
+- `bitcoind_rpc` depends on `bitcoind` being installed and available in `PATH` or in the `BITCOIND_EXEC` ENV variable.
+- `esplora` depends on [`esplora_a33e97e1` version of `electrs`](https://github.com/Blockstream/electrs/tree/a33e97e1a1fc63fa9c20a116bb92579bbf43b254)
+   being installed and available in `PATH` or in the `ELECTRS_EXEC` ENV variable.
+
```

Comment:
We are removing the auto-download feature of `bitcoind` and `electrsd` in the `bitcoind_rpc` and `esplora` crates. I added an explanation in the repo and crates' `README.md`s. This also simplifies a little bit the MSRV pinning of deps.